src/core/token/TokenManager.ttslua

```diff
@@ -231,7 +231,7 @@ do
 
     -- this is used to load the correct state for additional resource tokens (e.g. "Ammo")
     local callback = nil
-    if tokenType == "resource" and stateID ~= nil then
+    if tokenType == "resource" and stateID ~= nil and stateID ~= 1 then
```

Comment:
Up to you if it's worth changing, but looking at this again now I think it would have been better to pass the type string all the way to this level, then convert it to the state number just before spawning (or even in the callback method).  Doing it at a higher level (and taking the state number here) means that the conversion code and state translation table are duplicated, and they're currently done so inconsistently (compare Playmat:550 to TokenManager:111).

Approving, but I think this would be worth cleaning up before submit.

Reply:
Valid objection, I'll change that

Reply:
@Buhallin I've updated it, but can't get rid of the table in the playmat since I need to check the memo of the tokens. Are you fine with merging it like this?

Reply:
Finale update, moved the replenishing function to token manager to avoid repetion